src/test/java/de/tum/in/www1/artemis/util/DatabaseUtilService.java

```diff
@@ -516,8 +516,8 @@ public Exam addExam(Course course) {
         return exam;
     }
 
-    public ExerciseGroup addExerciseGroup(Exam exam) {
-        ExerciseGroup exerciseGroup = ModelFactory.generateExerciseGroup(exam);
+    public ExerciseGroup addExerciseGroup(Exam exam, boolean mandatory) {
```

Comment:
Just a question because I did it differently in my PR: You never really use `mandatory` right? It is always set to `true` in your test cases?

Reply:
Yes but this might be useful when mandatory/optional feature is implemented

Reply:
Ok then the merge order of your and my PR will decide whether this will survive :-D